ds-prep-python-sets/main.py

```diff
@@ -0,0 +1,48 @@
+# 3
+trail_mix = set({'m&ms', 'peanuts', 'raisins'})
```

Comment:
The `set` functions call is unnecessary for making a `set` when using curly brace syntax: https://www.w3schools.com/python/python_sets.asp

---

![image](https://user-images.githubusercontent.com/16143771/148819244-34a41556-cdec-4851-84ae-9abc7702ad4d.png)